Makefile

```diff
@@ -0,0 +1,180 @@
+# Copyright 2019 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+###############
+# Dev Helpers #
+###############
+SHELL := /bin/bash
+GOBIN ?= $$PWD/bin
+GOOS ?= linux
+define build
+    mkdir -p $(GOBIN) && \
+		GOOS=$(GOOS) \
+		GOARCH=amd64 \
+		CGO_ENABLED=0 \
+		GOBIN=$(GOBIN) \
+		GO111MODULE=on \
+		go install -v \
+		-gcflags='-e' \
+		$(1)
```

Comment:
Did you determine that `-installsuffix cgo` was not necessary?

Reply:
https://plus.google.com/117192131596509381660/posts/eNnNePihYnK